passivbot.py

```diff
@@ -810,11 +810,23 @@ def make_get_filepath(filepath: str) -> str:
 
 def load_key_secret(exchange: str, user: str) -> (str, str):
     try:
-        return json.load(open(f'api_key_secrets/{exchange}/{user}.json'))
-    except(FileNotFoundError):
-        print(f'\n\nPlease specify {exchange} API key/secret in file\n\napi_key_secre' + \
-              f'ts/{exchange}/{user}.json\n\nformatted thus:\n["Ktnks95U...", "yDKRQqA6..."]\n\n')
-        raise Exception('api key secret missing')
+        keyfile = json.load(open('api-keys.json'))
+        # Checks that the user exists, and it is for the correct exchange
+        if user in keyfile and keyfile[user]["exchange"] == exchange:
+
+            # If we need to get the `market` key:
```

Comment:
It will confuse more than it will give guidance to speak about a configurable market that is not yeat configurable, commented code is "dead code" and didn't give any information of the intention :)
What i suggest you is to remove the lines in api-keys.json and add a comment in this file with a link of a message from the discussion of this PR, if we need to add the feature of a configurable market we will inevitably see this comment left in the code :)